pom.xml

```diff
@@ -1048,9 +1048,9 @@
             </dependency>
 
             <dependency>
-                <groupId>commons-collections</groupId>
-                <artifactId>commons-collections</artifactId>
-                <version>3.2.2</version>
+                <groupId>org.apache.commons</groupId>
```

Comment:
@ptirador,

I think this will require a bit more work, such as applying exclusions for all dependencies that transitively depend on the old artifact -- `commons-collections`, which you can check easily:

```
carlspring@carlspring:/java/opensource/carlspring/strongbox-clean> mvn dependency:tree > dep.tree
carlspring@carlspring:/java/opensource/carlspring/strongbox-clean> cat dep.tree | grep collections
[INFO] |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |     +- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |     +- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
[INFO] |  |  |  \- commons-collections:commons-collections:jar:3.2.2:compile
```

As you can see, there's quite a few of them and you'll need to manually add an `<exclusion/>` for each of them in both the `strongbox-parent` and `strongbox-proto-parent` projects.

Reply:
This have been done in `strongbox` project. Please look at the PR. There is no transitive dependencies for this project.